frontend/modules/shared/ui/src/lib/layout/top-bar/top-bar.component.html

```diff
@@ -10,6 +10,8 @@
   <ng-content></ng-content>
 </div>
 
+<reunice-language-switcher></reunice-language-switcher>
```

Comment:
I think you can move it down next to profile dropdown, the whole right section is meant to add more dropdowns/menus/buttons, the middle one is for search bar. Probably you'll face issue with tui-select to become really small, you can add `min-width` in the styles for `:host` later